apps/search/src/app/app.component.ts

```diff
@@ -1,15 +1,41 @@
-import { Component } from '@angular/core'
-import { ColorService } from '@lib/common'
+import { Component, OnInit } from '@angular/core'
+import { BootstrapService, ColorService } from '@lib/common'
+import {
+  RequestMoreResults,
+  SearchState,
+  SetConfigAggregations,
+} from '@lib/search'
+import { Store } from '@ngrx/store'
+import { map, pluck, take, tap } from 'rxjs/operators'
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'search'
 
-  constructor() {
+  constructor(
+    private bootstrap: BootstrapService,
+    private store: Store<SearchState>
+  ) {
     ColorService.applyCssVariables('#e73f51', '#c2e9dc', '#212029', '#fdfbff')
   }
+
+  ngOnInit(): void {
+    this.bootstrap
+      .uiConfReady('srv')
+      .pipe(
+        take(1),
+        map((config) => config.mods.search.facetConfig),
+        // TODO: make the config work not just for tag
+        pluck('tag'),
+        tap((tagConfig) => {
+          this.store.dispatch(new SetConfigAggregations({ tag: tagConfig }))
```

Comment:
Careful here, `tag` has been changed to `tag.default` somehow.

Reply:
The field name change is related to https://github.com/geonetwork/core-geonetwork/pull/5193.
Why only adding the tag here ? it should be an array of facet as defined in the config no?

Reply:
Yes, see my TODO comment just above. For now, the aggregations configuration can't be used as is. There are some property to remove concerning roles and some other stuff.

Reply:
> Why only adding the tag here ? it should be an array of facet as defined in the config no?

I just moved code. The facet configuration is handled in other PRs.